### PR TITLE
BUG: fix unmatched dtype between y_train and y_test

### PR DESCRIPTION
**Pull request by facaiy**
y_train_np is uint8, while y_test_np is int64:

```python
from tensorflow.python.keras.datasets import cifar10
(x_train_np, y_train_np), (x_test_np, y_test_np) = cifar10.load_data()
print(y_train_np.dtype, y_test_np.dtype)
# uint8 int64
```

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ekelsen: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ekelsen: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @ekelsen: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
